### PR TITLE
Documentation Badge Added

### PR DESCRIPTION
**Pull request by DEKHTIARJonathan**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -220,3 +220,8 @@ DALI was built with major contributions from Trevor Gale, Przemek Tredak, Simon
 
 .. |License| image:: https://img.shields.io/badge/License-Apache%202.0-blue.svg
    :target: https://opensource.org/licenses/Apache-2.0
+
+.. |Documentation| image:: https://img.shields.io/badge/nvidia%20dali-documentation-brightgreen.svg?longCache=true
+   :target: https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/
```
Comment (JanuszL) [inline README.rst]: Maybe we should point to https://docs.nvidia.com/deeplearning/sdk/dali-master-branch-user-guide/docs/, as it is master branch?
@ptrendx ?

**Comment by DEKHTIARJonathan** [comment]
@januszl usually the documentation is supposed to point to the last stable release. However, as most people probably compile DALI from source maybe we can go as you suggested.

Tell me which link you would like and I will make the correction.

**Comment by JanuszL** [comment]
@DEKHTIARJonathan  - you are right, TensorFlow does that. So I will follow your suggestion.

File: README.rst (hunk 1 above)
Comment (JanuszL) [reply README.rst]: BTW - it should be `Nvidia DALI` I believe.

Comment (DEKHTIARJonathan) [reply README.rst]: image

@JanuszL done

**Pull request merged**